app/domain/appeal.ts

```diff
@@ -0,0 +1,5 @@
+export interface Appeal {
```

Comment:
Can we move this to types folder to have all the interfaces and types in one place?